sdk/storage/azure-storage-blob/azure/storage/blob/blob_service_client.py

```diff
@@ -40,8 +40,10 @@
         Metrics,
         RetentionPolicy,
         StaticWebsite,
-        CorsRule
+        CorsRule,
+        PublicAccess
     )
+SnapshotProperties = TypeVar("SnapshotProperties")
```

Comment:
SnapshotProperties is not longer a type - this should be removed and if you see a reference to "SnapshotProperties" it's now just a `dict[str, Any]`